app/controllers/api/v1/file_pushes_controller.rb

```diff
@@ -124,6 +125,7 @@ def show
 
       {
         "url_token": "quyul5r5w18",
+        "name": null,
```

Comment:
These attributes in responses are added as default. If it is necessary to show them when necessary, I need to `to_json` methods of models. Do you want me to update it?

Reply:
Good point - `name` should only be shown in `to_json` if the requesting user is authenticated and is the owner.  Same with `note` but I believe this is already enforced.

Reply:
[It is updated.](https://github.com/pglombardo/PasswordPusher/pull/3351#issuecomment-2879406671)